features/dd-sdk-android-trace-internal/src/test/kotlin/com/datadog/trace/core/propagation/BaggageTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+package com.datadog.trace.core.propagation
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+
+internal class BaggageTest {
+
+    @ParameterizedTest
+    @MethodSource("invalidBaggageHeaderSamples")
+    fun `M return empty W from { invalid header value }`(
+        headerValue: String?
+    ) {
+        assertThat(Baggage.from(headerValue).asMap()).isEqualTo(emptyMap<String, String>())
```

Comment:
probably there is `isEmpty` for `Map`?

```suggestion
        assertThat(Baggage.from(headerValue).asMap()).isEmpty()
```